docs/opensafely-cli.md

```diff
@@ -194,39 +194,56 @@ opensafely exec IMAGE bash
 
 This can be useful if you want to explore the image manually.
 
+### `launch` - Run web based interactive tools
 
-### `rstudio` - Running RStudio Server
+Many users may prefer to use some of the interactive development environments that are commonly used for different languages, like Rstudio or Jupyter Lab.
 
-An alternative method to open your project in an RStudio Server session is to open your research repository in [Codespaces](/getting-started/how-to/use-github-codespaces-in-your-project/#how-to-access-rstudio).
+Using `opensafely launch`, you can run these tools from the OpenSAFELY images, accessing them via your browser. This means that code you write and run in these tools will be running the OpenSAFELY provided environment, with the right dependencies and versions, rather than what ever you have installed on your machine.
```

Comment:
```suggestion
Using `opensafely launch`, you can run these tools from the OpenSAFELY images, accessing them via your browser. This means that code you write and run in these tools will be running the OpenSAFELY-provided environment, with the right dependencies and versions, rather than whatever you have installed on your own machine.
```